.vsts-ci/install-ps.yml

```diff
@@ -48,19 +48,31 @@ phases:
     jobName: InstallPowerShellUbuntu
     pool: ubuntu-latest
     verification: |
-      if ([Version]"$($PSVersionTable.PSVersion.Major).$($PSVersionTable.PSVersion.Minor).$($PSVersionTable.PSVersion.Patch)" -lt [version]"6.2.0")
+      if ([Version]"$($PSVersionTable.PSVersion.Major).$($PSVersionTable.PSVersion.Minor).$($PSVersionTable.PSVersion.Patch)" -lt [version]"7.3.0")
```

Comment:
I _think_ this can be expressed as:
`if([Version]$PSVersionTable.PSVersion -lt "7.3.0")`
we did some stuff to enable converting a semanticversion into a version, but it's fine as is